### ISSUE TITLE
old messages appear as new ones

### ISSUE THREAD

**Issue by aaannndddyyy** [Antox/Antox#357]
I have twice received Antox new message notifications when Antox was not even connected to the internet, and new messages with current timestamp were shown, but those messages were actually old messages that had been received weeks ago. 
Being shown as new and latest though. 
Am using latest fdroid version.

**Comment by subliun** [comment]
Duplicate of #282

**Comment by aaannndddyyy** [comment]
No, This is not a duplicate of 282.
This issue here is not about faux offline messages.
You must have been in a hurry and misread this issue.

**Comment by subliun** [comment]
You're right. Oops. Sorry about that. So there aren't actually new messages shown just new message notifications?
